test/accountSpec.js

```diff
@@ -689,6 +686,88 @@ describe('Accounts', function () {
       })
     })
 
+    it('should resend notifications when asked to', function * () {
+      const listener = sinon.spy()
+      this.socket.on('message', (msg) => listener(JSON.parse(msg)))
+
+      const transfer = this.transfer
+
+      yield this.request()
+        .put(transfer.id)
+        .auth('alice', 'alice')
+        .send(transfer)
+        .expect(201)
+        .expect(validator.validateTransfer)
+        .end()
+
+      // TODO: Is there a more elegant way?
+      yield timingHelper.sleep(50)
+
+      const transferId = transfer.id.match('http://localhost/transfers/(.+)')[1]
+      this.socket.send(JSON.stringify({ type: 'request_notification',
+                                        id: transferId }))
+
+      yield timingHelper.sleep(50)
+
+      sinon.assert.calledTwice(listener)
```

Comment:
There is an [addTransfers](https://github.com/interledger/five-bells-ledger/blob/master/test/helpers/db.js#L42) helper method. I think it's clearer to have two tests, for the separation of logic, but this test encompasses both cases, so I don't think this issue needs to hold up the PR.